pkg/manager/manager.go

```diff
@@ -240,10 +241,11 @@ func (m *Manager) CheckCacheForDevice(target devicetype.ID, deviceType devicetyp
 	version devicetype.Version) (devicetype.Type, devicetype.Version, error) {
 
 	deviceInfos := mgr.DeviceCache.GetDevicesByID(target)
-	topoDevice, errTopoDevice := mgr.DeviceStore.Get(topodevice.ID(target))
+	object, errTopoDevice := mgr.DeviceStore.Get(topo.ID(target))
 	if errTopoDevice != nil {
 		log.Infof("Device %s not found in topo store", target)
```

Comment:
probably should throw error here - no point continuing without `object`

Reply:
done